index.html

```diff
@@ -125,7 +109,6 @@
   <script src="scripts/view/projectView.js"></script>
```

Comment:
Remember to stop including githubToken.js in your HTML now you're not using it anymore.